docs/pages/SAPUI5.md

```diff
@@ -1,3 +1,68 @@
 # Consuming SAPUI5 libraries
 
-Coming soon
+!!! info
+    **Make sure you have installed the latest UI5 CLI in version 2.0 or later:**
```

Comment:
Make sure you have installed the latest UI5 CLI in Version 2.0 or later

Reply:
Done